### PR TITLE
Feat ✨: An Endpoint to send video to user's email

### PR DESCRIPTION
**Pull request by DestinedCodes**
​
## Description

​I added an endpoint that uses smtp library to send email from the helpmeout.hngx@gmail.com account to the email address of users 

## Related Issue (Link to linear ticket)

[Linear Ticket](https://linear.app/zuri-project-backend/issue/HEL-1/create-video-disbursing-endpoint)
## Motivation and Context

​This change is required to enhance the functionality of the application by allowing users to send videos to their email addresses, making it more convenient for them to access their recordings.

## How Has This Been Tested?

​I have thoroughly tested this feature to ensure it functions correctly. It includes testing in different environments and ensuring it doesn't affect other parts of the code.

## Screenshots (if appropriate - Postman, etc):
Screenshot_20231012-071422
Screenshot_20231012-071406

​
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
​
## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by DestinedCodes** [comment]
The email template us still being worked on by a FE dev, but all other fucntionalities are okay.

File: app/services/mail_service.py [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    msg["To"] = recepient_address
+    msg.set_content(get_mail(video_id), subtype="html",)
+
+    with smtplib.SMTP(EMAIL_HOST, EMAIL_PORT) as smtp:
+        smtp.starttls(context=ssl.create_default_context())
+        smtp.login(EMAIL_ADDRESS, EMAIL_PASSWORD)
+        smtp.send_message(msg)
+
+
+def get_mail(video_id):
```
Comment (zxenonx) [inline app/services/mail_service.py:35]: type hint missing here

**Review by zxenonx — CHANGES_REQUESTED** [review]
Good. @DestinedCodes 
Also please gitignore the mp4 file

**Review by zxenonx — APPROVED** [review]
Thanks

**Pull request merged**